tests/unit/cpp_api.cpp

```diff
@@ -560,3 +560,14 @@ TYPED_TEST(CppApiTestFixture, TestMapErase) {
   map.Erase("2");
   ASSERT_EQ(map.Size(), 0);
 }
+TYPED_TEST(CppApiTestFixture, TestNodeRemoveProperty) {
+  mgp_graph raw_graph = this->CreateGraph(memgraph::storage::View::NEW);
+  auto graph = mgp::Graph(&raw_graph);
+  auto node = graph.CreateNode();
+
+  int64_t int1 = 100;
+  mgp::Value value = mgp::Value(int1);
+  node.SetProperty("key", value);
```

Comment:
Can you add one assert in the middle between setting and removing that we have actually set the property?

Reply:
So, size==1, or something like that